src/_how_tos/sw-cli.md

```diff
@@ -68,7 +69,7 @@ For those who are curious, let's look at what's in the
 generated service worker.
 
 ```
-importScripts('sw-lib.v0.0.13.min.js');
+importScripts('workbox-sw.prod.v0.0.2.js');
```

Comment:
Folks are likely to copy/paste whatever we provide in the docs here. Silly question, but can we pipe in the latest version of the package and use templating to populate the version part of this import?